Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -76,6 +76,11 @@ configuration:
   - Incoming And Outgoing
   required: false
   type: 15
+- additionalinfo: Select this checkbox to retrieve comments and work notes without accessing the `sys_field_journal` table.
```

Comment:
we will probably need more documentation in the readme about when this is needed

Reply:
I added some more details in line 2490 in the readme, but I also asked Jane Rosen to help us here so it will be clear enough.